vscode/webviews/Components/FileLink.tsx

```diff
@@ -46,7 +49,36 @@ export const FileLink: React.FunctionComponent<FileLinkProps> = ({
                 getVSCodeAPI().postMessage({ command: 'openFile', uri, range })
             }}
         >
+            {metadata && <MetadataElement metadata={metadata} />}
             {pathWithRange}
         </button>
     )
 }
+
+// Pretty-print the score, so that at most 2 sig figs are used, and if a number
+// is greater than one million, 'M' is used, and if a number is between one
+// thousand and one million, 'K' is used
+function prettyPrintScore(score: number): string {
+    if (score > 1000000) {
```

Comment:
Style nit, you could use _s here too for consistency.
```suggestion
    if (score > 1_000_000) {
```